internal/pkg/show/config/plugin/argocd.go

```diff
@@ -0,0 +1,34 @@
+package plugin
+
+var ArgocdDefaultConfig = `tools:
+- name: argocd
+  # name of the plugin
+  plugin: argocd
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: argo
+      # url of the Helm repo
+      url: https://argoproj.github.io/argo-helm
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: argo/argo-cd
+      # release name of the chart
+      release_name: argocd
+      # k8s namespace where ArgoCD will be installed
+      namespace: argocd
+      # whether to wait for the release to be deployed or not
+      wait: true
+      # the time to wait for any individual Kubernetes operation (like Jobs for hooks). This defaults to 5m0s
+      timeout: 5m
+      # whether to perform a CRD upgrade during installation
+      upgradeCRDs: true
+      # custom configuration (Optional). You can refer to [ArgoCD values.yaml](https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml)
+      values_yaml: |
+        controller:
+          service: 
+            port: 8080
+`
```

Comment:
We should remove the "\n" after "8080", otherwise the output has an extra new line:

<img width="503" alt="Screen Shot 2022-04-07 at 11 24 09" src="https://user-images.githubusercontent.com/19277614/162114031-6965f03d-4534-428c-9c3b-36346e11adbc.png">

Reply:
got it